astropy/__init__.py

```diff
@@ -239,3 +239,16 @@ def online_help(query):
         del locals()[varname]
 
 del varname, __module_type__
+
+
+def flpr():
```

Comment:
Great work! :+1:

Prayers to the gods of computing occasionally take a long time to be answered.  Can I suggest we make this an asynchronous function?
```suggestion
async def flpr():
```